pkg/repository/repository.go

```diff
@@ -393,7 +393,6 @@ func (r *repository) getObject(ctx context.Context, name string, namespace strin
 	)
 	if err != nil {
 		namespacedName := getNamespacedName(name, namespace)
-		log.Error(err, "failed to get object from api server", "object", namespacedName)
```

Comment:
it might still be worth logging the error when the error is not a NotFound error